src/components/backgroundselector/BackgroundService.js

```diff
@@ -45,7 +46,12 @@ goog.require('ga_permalink');
       };
 
       var getBgByTopic = function(topic) {
-        var topicBg = getBgById(topic.defaultBackground) || bgs[0];
+        var topicBg = null;
+        if (topic.plConfig && topic.plConfig.length) {
```

Comment:
to avoid testing `.length` . I suggest you set the proper value when we parse the topic onfiguration in https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/topic/TopicService.js#L22